### DISCUSSION TITLE
Co-pilot for AI reporter

### DISCUSSION THREAD

**Discussion by wliew99** [ctrf-io/ctrf#21] [Ideas]
For AI reporting feature, can co-pilot be used instead of ChatGPT?

**Comment by Ma11hewThomas** [comment]
Hey @wliew99, 

I'm not sure this is possible, I understand Copilot is designed to be used within Microsoft's own products rather than integrated into external applications via APIs. If I've misunderstood, please let me know.

I'm happy to assess adding new models for the AI test reporter should they expose their services via APIs.
